src/runtime_src/driver/xclng/tools/xbutil/xbutil.h

```diff
@@ -207,32 +211,53 @@ class device {
         return 0;
     }
 
+    int parseComputeUnits(std::vector<ip_data> &computeUnits) const
```

Comment:
Maybe we should stop adding implementation code in header file...

Reply:
I agree, I will try to move this into the sensor files but that's beyond the scope of this feature.